evennia/contrib/actions/README.md

```diff
@@ -0,0 +1,313 @@
+# The action system
```

Comment:
Trying to read this file in an actual code editor made it clear the lines here are immensely long, making it very hard to read. My editor is 150 characters wide and it's nowhere wide enough. Please shorten the line length to something like 80-100 characters to make it be readable by most normal editors (see existing contribs for examples).